.github/scripts/compare-images-and-pdfs.py

```
@@ -0,0 +1,455 @@
+            raise ValueError("Missing required environment variables: GITHUB_TOKEN, GITHUB_REPOSITORY, PR_NUMBER")
+
+        self.headers = {
+            'Authorization': f'token {self.github_token}',
+            'Accept': 'application/vnd.github.v3+json'
+        }
+        self.base_url = f'https://api.github.com/repos/{self.repo}'
+
+    def get_pr_files(self) -> List[Dict[str, Any]]:
+        """Get list of changed files in the PR"""
+        url = f'{self.base_url}/pulls/{self.pr_number}/files'
+        response = requests.get(url, headers=self.headers)
```

Ruff (0.15.20)

[error] Probable use of `requests` call without timeout

(S113)